includes/class-wc-gateway-payfast.php

```diff
@@ -1203,24 +1258,30 @@ public function cancel_subscription_listener( $subscription ) {
 	}
 
 	/**
+	 * Cancel a pre-order subscription.
+	 *
 	 * @since 1.4.0
-	 * @param string $token
 	 *
-	 * @return bool|WP_Error
+	 * @param string $token Payfast subscription token.
+	 *
+	 * @return bool|WP_Error WP_Error on failure, bool true on success.
 	 */
 	public function cancel_pre_order_subscription( $token ) {
 		return $this->api_request( 'cancel', $token, array(), 'PUT' );
 	}
 
 	/**
+	 * Generate the parameter string to send to Payfast.
+	 *
 	 * @since 1.4.0 introduced.
-	 * @param      $api_data
-	 * @param bool $sort_data_before_merge? default true.
-	 * @param bool $skip_empty_values Should key value pairs be ignored when generating signature?  Default true.
+	 *
+	 * @param array $api_data               Data to send to the Payfast API.
+	 * @param bool  $sort_data_before_merge Whether to sort before merge. Default true.
+	 * @param bool  $skip_empty_values      Should key value pairs be ignored when generating signature? Default true.
 	 *
 	 * @return string
 	 */
-	protected function _generate_parameter_string( $api_data, $sort_data_before_merge = true, $skip_empty_values = true ) {
+	protected function _generate_parameter_string( $api_data, $sort_data_before_merge = true, $skip_empty_values = true ) {  // phpcs:ignore PSR2.Methods.MethodDeclaration.Underscore
```

Comment:
Wondering if we should globally disable this rule since we're having to ignore that on multiple lines within this file?

Reply:
> Wondering if we should globally disable this rule since we're having to ignore that on multiple lines within this file?

Done in aac4e4db25f3bb290f1ee55edfb983cff2e52b90. I excluded the rule for this file only as none of the other files were affected.